docs/architecture/process-view.md

```diff
@@ -0,0 +1,80 @@
+# Process View
+
+* Solution architect
+
+
+
+* Sequence
+* Communication
+* Activity
+* Timing
+* Interaction overview
+
+
+
+<figure><img src="../.gitbook/assets/image.png" alt=""><figcaption><p><a href="https://docs.axoniq.io/reference-guide/architecture-overview">https://docs.axoniq.io/reference-guide/architecture-overview</a></p></figcaption></figure>
```

Comment:
This pic needs explanation

Reply:
> The image below shows a typical architecture of an Axon based application.
In such an architecture, a UI (or API) can send commands to request to change an application's state. These Commands are handled by a Command Handling Component, which uses a model to validate the command and make decisions on which side-effects to trigger (if any).
The side-effects caused by commands are published using events. These events are persisted into the Event Store, after which they become available to the Event Handling Components. These will, in turn, take their appropriate action.
A typical action is updating the view models, which for example allow the UI to render the application's state. Other actions could be sending messages to external components, or even triggering other side-effects through new commands.
To retrieve the required information from the query model, the UI (or API) can send queries to their respective Query Handling Components. The Query Handling Components in turn respond by returning sections of the query model as dedicated query responses.
The separation of the Command Model and the Query Models (also called View Models or Projections) allows these models to only focus on that specific aspect of the application. This makes each individual model easier to comprehend, and therefore more maintainable in the long term.